client/src/app/employee/employee-detail/employee-detail.component.html

```diff
@@ -81,7 +81,7 @@ <h5>Employee Details</h5>
       [payrolls]="employee.payrolls"
     ></app-employee-payroll-list>
   </div>
-  <div class="col-3 pt-3">
+  <div class="col-3 pt-3" *ngIf="employee.status === 'Enabled'">
```

Comment:
This is an example of there either a function or class could encapsulate this nicely.

```typescript
class Employee {
  ...

  isEnabled() {
    return this.status === 'Enabled';
  }
}
```

Reply:
I like this over checking again hardcoded values

Reply:
i like that idea over using hardcoded values